bci_tester/data.py

```diff
@@ -161,6 +162,8 @@ def _get_repository_name(image_type: _IMAGE_TYPE_T) -> str:
         return ""
     if TARGET == "ibs-cr":
         return "images/"
+    if TARGET == "factory-totest":
+        return "containers/opensuse/bci/"
```

Comment:
Are you sure about this? This only makes sense if all images are released into the `containers` namespace in ToTest.

Reply:
AFAICT this applies for `init, minimal, micro, busybox, golang, rust, ruby and postgres14` images that I have checked.